nicoRDL/dotTheory-JS
# JS-202  (JS-352) JavaScript and the DOM: Making Changes in the DOM

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-352

## Description

- completed code challenges on adding, inserting, and changing html element styles through the DOM
- completed all quizzes

## Steps to Validate
1. Open the index.html
2. Observe the text field; the following paragraph element should display the text inside the field

## Risks
N/A
